src/components/Navbar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Button from '@material-ui/core/Button';
+import { isLoggedIn } from '../helpers/application_helper';
+import { makeStyles } from '@material-ui/core/styles';
+import AppStyles from '../assets/jss/styles/App.styles.jsx'
+// eslint-disable-next-line import/no-webpack-loader-syntax
```

Comment:
If I'm not mistaken we aren't using eslint, so it can be removed

Reply:
seems like I use it in vs code